### ISSUE TITLE
Additional (background) data:

### ISSUE THREAD

**Issue by edupenabad** [jonblower/hazards-lab#7]
There are some ideas about the background information that could be combined with tracks:
	* socio-economic data: GDP, losses, crops, etc
	* sea surface temperature: climatology (climatic? yearly? monthly?), direct values or anomalies?
	* Cyclone Damage Potential index (Holland and Done: http://www2.mmm.ucar.edu/people/done/files/CDP.pdf)
	* Track density: some investigation needs to be done here. 
		- How we calculate density? (gridding, smoothing, etc): according to Kevin (via Alison): "track densities are computed using the kernel estimator method with spherical kernels"
		- Could GIS capabilities be useful for calculate line densities?

**Comment by edupenabad** [comment]
Update (after 28.07.2015 meeting)

About population data:

Alison suggested some sources
http://sedac.ciesin.columbia.edu/maps/gallery/search
http://www.globalquakemodel.org/what/physical-integrated-risk/exposure-database/

Provide population density data, ideally as NetCDF but could also be as an image, or shapefiles (Alison)


About track density:
Eduardo calculates track densities (make sure to compute them "using the kernel estimator method with spherical kernels"
(look issue #11 about averaging period)

**Comment by edupenabad** [comment]
I have checked this morning the SEDAC site suggested by Alison for population data, and they have a Geoserver with some web services enabled containing some interesting layers that can be directly displayed with our current tool (using Leaflet.js WMS capabilities).

In any case it is still interesting to have access to the data we were looking for, in order to display them more properly and have them available for "calculations", but using the WMS of the SEDAC server we can have a first initial overview of those data plotting them directly over our maps.

In summary, in the current version of the web tool that you can access from the UoR's network, I have introduced some of that SEDAC layers. They can be selected from the "layer control" in the upper right corner

**Comment by jonblower** [comment]
Good idea. It will be an interesting GUI problem to link datasets with different averaging periods (e.g. we might want to show storms for a season, but we might only have population data on a yearly basis).

**Comment by AlisonCobb** [comment]
Dear both,


I have heard from Ben L-H and he says:



Nigel is expecting to receive updated SSP [ http://sedac.ipcc-data.org/ddc/ar5_scenario_process/parallel_nat_scen.html ]
 data shortly. Meanwhile, if you’re in a rush, we’ve got some older data on a 0.5x0.5 degree grid that we can package up. Do you have a preferred data format?


Jon, these are the new scenarios that Nigel was talking about when we met a few months ago.


With regards to the preferred format, what are your thoughts?


Many thanks,
Alison








On 6 Aug 2015, at 13:45, Eduardo Penabad Ramos <notifications@github.com> wrote:


I have checked this morning the SEDAC site suggested by Alison for population data, and they have a Geoserver with some web services enabled containing some interesting layers that can be directly displayed with our current tool (using Leaflet.js
 WMS capabilities).
In any case it is still interesting to have access to the data we were looking for, in order to display them more properly and have them available for "calculations", but using the WMS of the SEDAC server we can have a first initial overview of
 those data plotting them directly over our maps.
In summary, in the current version of the web tool that you can access from the UoR's network, I have introduced some of that SEDAC layers. They can be selected from the "layer control" in the upper right corner
—
Reply to this email directly or 
view it on GitHub.











Alison Cobb
Research Fellow

NCAS-Climate
Department of Meteorology
University of Reading 
Earley Gate
PO Box 243
Reading RG6 6BB
tel: +44(0)118 378 6238

National Centre for Atmospheric Science:
www.ncas.ac.uk

**Comment by jonblower** [comment]
CF-compliant NetCDF would be the best format, but others are possible too (we can probably convert). By the way, you might want to edit your post above to remove the whitespace and your signature (GitHub does not seem to have stripped them out).

**Comment by AlisonCobb** [comment]
Thanks - edited!
I will get back to Ben with the above.

**Comment by edupenabad** [comment]
SSP-Population is currently being showed in the WCHL

They have been also added as background fields the "timefilter-dependant" densities for every dataset (these are "surrogate" values, roughly calculated as 2D histograms, more work on this should be done, look at issue #11 )
